.github/workflows/run-tests.yml

```diff
@@ -10,17 +10,28 @@ jobs:
 
     steps:
       - uses: actions/checkout@v3
-      - name: Set up Python 3.10
+
+      - name: Set up Python 3.12
         uses: actions/setup-python@v4
         with:
-          python-version: '3.10'
+          python-version: '3.12'
+
       - name: Set up Node
         uses: actions/setup-node@v3
         with:
           node-version: 18
-      - run: npm install
-      - run: npm install -g lighthouse
+
+      - run: npm ci
+
+      - run: npm install -g lighthouse@11.7.1
```

Comment:
To get the tests passing, I have pinned us to an older version of lighthouse ( https://github.com/DemocracyClub/design-system/commit/481d95523c9ba94742945fdd8dd7d8487b159744 was pushed just after the release of lighthouse 12). Lets address the lighthouse 11 --> lighthouse 12 migration as another issue.

Reply:
https://github.com/DemocracyClub/design-system/pull/142